.github/workflows/uml_action.yml

```diff
@@ -28,4 +28,4 @@ jobs:
           author_name: GitHub Action
           message: "GitHub action UML generation auto-update"
           add: "./doc/source/user_guide/documentation/*"
-          ref: git branch --show-current
+          branch: git branch --show-current
```

Comment:
Looking at the [EndBug/add-and-commit changelog](https://github.com/EndBug/add-and-commit/releases/tag/v8.0.0), it seems like v8 works directly on the branch that was checked out so this line won't be needed anymore. Could you remove this line, and change L26 from `uses: EndBug/add-and-commit@v7` to `uses: EndBug/add-and-commit@v8` and see if it fixes things?